Cargo.toml

```diff
@@ -12,6 +12,7 @@ license = "MIT"
 readme = "README.md"
 
 [dependencies]
+layout-macro = { version = "0.1.0", path = "./layout-macro" }
```

Comment:
as it will be exported to crates.io independently, better to prefix it with keyberon, to be clear that's an internal depedency.

and better naming this crate simply 'keyberon-macros', as there might be other macros later.